doc/content/source/solvers/MFEMCGSolver.md

```diff
@@ -0,0 +1,15 @@
+# MFEMCGSolver
+
+## Summary
+
+!syntax description /Solver/MFEMCGSolver
+
+## Overview
+
+Defines and builds an `mfem::CGSolver` solver to use as an iterative solver to solve the MFEM equation system.
```

Comment:
For this and the MFEM GMRES solver, it might be worth it to mention that they are compatible with partially-assembled systems. Alternatively, we could mention in the Hypre solvers that they are *not* compatible with partial assembly.

Reply:
Added